frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/VariantDeleteDialog/VariantDeleteDialog.tsx

```diff
@@ -0,0 +1,42 @@
+import { Alert, styled } from '@mui/material';
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import { IFeatureVariant } from 'interfaces/featureToggle';
+
+const StyledLabel = styled('p')(({ theme }) => ({
+    marginTop: theme.spacing(3),
+}));
+
+interface IVariantDeleteDialogProps {
+    variant?: IFeatureVariant;
+    open: boolean;
+    setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    onConfirm: () => void;
+}
+
+export const VariantDeleteDialog = ({
+    variant,
+    open,
+    setOpen,
+    onConfirm,
+}: IVariantDeleteDialogProps) => {
+    return (
+        <Dialogue
+            title="Delete variant?"
+            open={open}
+            primaryButtonText="Delete variant"
+            secondaryButtonText="Close"
+            onClick={onConfirm}
+            onClose={() => {
+                setOpen(false);
+            }}
+        >
+            <Alert severity="error">
+                Deleting this variant will change which variant users receive.
+            </Alert>
+            <StyledLabel>
+                You are about to delete variant:{' '}
+                <strong>{variant?.name}</strong>
+            </StyledLabel>
+        </Dialogue>
+    );
+};
```

Comment:
Pretty much an updated duplicate of `frontend/src/component/feature/FeatureView/FeatureVariants/FeatureVariantsList/useDeleteVariantMarkup.tsx`.